designer/server/src/test/scala/pl/touk/nussknacker/ui/api/NuDesignerApiAvailableToExposeYamlSpec.scala

```diff
@@ -41,9 +50,15 @@ class NuDesignerApiAvailableToExposeYamlSpec extends AnyFunSuite with Matchers {
   }
 
   test("Nu Designer OpenAPI document with all available to expose endpoints has to be up to date") {
-    val currentNuDesignerOpenApiYamlContent =
-      (Project.root / "docs-internal" / "api" / "nu-designer-openapi.yaml").contentAsString
-    NuDesignerApiAvailableToExpose.generateOpenApiYaml should be(currentNuDesignerOpenApiYamlContent)
+    // todo NU-1772: OpenAPI differs when generated on Scala 2.12 and Scala 2.13 (order of endpoints is different)
```

Comment:
When are we supposed to fix it?

Reply:
Probably never, as we plan to removed Scala 2.12 cross compile

Comment changed to:
```
    // OpenAPI differs when generated on Scala 2.12 and Scala 2.13 (order of endpoints is different).
    // - test is ignored on Scala 2.12,
    // - it is probably not necessary to fix it, because we plan to remove Scala 2.12 support anyway
```